app/src/main/res/values/colors.xml

```diff
@@ -8,6 +8,7 @@
     <color name="statusBarColor">@*android:color/primary_device_default_settings_light
     </color>
     <color name="red">@*android:color/error_color_device_default_dark</color>
+    <color name="ic_launcher_background">@*android:color/accent_device_default_light</color>
```

Comment:
Do we want to do this?

Reply:
I think this way of supplying the resources doesn't work with gradle builds, right?

Reply:
Works fine on both Gradle and AOSP builds